site-src/enhancements.md

```diff
@@ -53,21 +76,69 @@ It is likely that throughout this process, you will discuss a variety of
 alternatives. Be sure to document all of these in the NPEP, and why we decided
 against them. At this stage, the NPEP should be targeting the "Implementable" stage.
 
-### 6. Standard: Make API changes
-With the NPEP marked as "Implementable", it is time to actually make the proposed changes in our API. 
+### 6. Experimental: Make API changes as "Experimental
+
+With the NPEP marked as "Implementable", it is time to actually make the proposed changes in our API.
 In some cases, these changes will be documentation
 only, but in most cases, some API changes will also be required.
 
-When updating NPEP from "Implementable" to "Standard", status update may be the 
+It is important that every new feature of the API is marked as "Experimental" when it is introduced. Within the API, we
+use <network-policy-api:experimental> tags to denote experimental fields. Within Golang packages (conformance tests,
+CLIs, e.t.c.) we use the experimental Golang build tag to denote experimental functionality.
+
+Some other requirements must be met before marking a NPEP Experimental:
+
+the graduation criteria to reach Standard MUST be filled out
+a proposed probationary period (see next section) must be included in the NPEP and approved by maintainers.
+
+When updating NPEP from "Implementable" to "Standard", status update may be the
 only required change, but also feel free to link any API, docs, etc. changes that
 were made as an implementation for a given NPEP.
 
 Before changes are released they MUST be documented. NPEPs that have not been
 both implemented and documented before a release cut off will be excluded from
 the release.
 
+#### Probationary Period
```

Comment:
maybe this should be a subsection of `6. Experimental: Make API changes as "Experimental` ?

Reply:
It is :) notice the heading size 👍